src/components/collection_voting_slab/MemberVoteSummary.js

```diff
@@ -28,7 +28,7 @@ export const MemberVoteSummary = (props) => {
       }
     }, [
       title,
-      h((showMemberVotes? ArrowDropUp : ArrowDropDown), {
+      h((showMemberVotes? ExpandLess : ExpandMore), {
```

Comment:
Updated the arrow drop up/dropdown that I made in a recent PR to be consistent with the dropup/dropdown added in this PR (looks better in my opinion).